crypto/evp/e_aes.c

```diff
@@ -1301,6 +1303,7 @@ static int aes_gcm_ctrl(EVP_CIPHER_CTX *c, int type, int arg, void *ptr)
         gctx->taglen = -1;
         gctx->iv_gen = 0;
         gctx->tls_aad_len = -1;
+        gctx->use_seq_no = 0;
```

Comment:
GH won't let me comment on line 1296, but since you're there put a blank line after the decl/assignment and before the switch.